app/views/components/blockEditor.html

```diff
@@ -21,7 +21,7 @@
   </div>
   <label>{{block.title}} <em class="required" ng-show="block.required">*</em></label>
   <ng-switch on="block.type">
-    <p ng-switch-when="paragraphContent" ng-bind="block.content" class="display-linebreaks"></p>
+    <p ng-switch-when="paragraphContent" ng-bind="block.content.paragraph" class="display-linebreaks"></p>
```

Comment:
@OzzieOrca We have fixed the issue as shown below, committed and pushed to branch:
```js
if ($scope.block.type === 'paragraphContent' &&
          angular.isDefined($scope.block.content) &&
          _.isString($scope.block.content)) {
          var prevValue = $scope.block.content;
          $scope.block.content = {
            paragraph: prevValue,
            ruleoperand: 'AND'
          };
        }
```